eng/pipelines/templates/pipeline.yml

```diff
@@ -1,233 +1,240 @@
 parameters:
-- name: DartLabEnvironment
-  displayName: DartLab Environment
-  type: string
-  default: Production
-  values:
-  - Production
-  - Staging
-- name: E2EPart1AgentCleanup
-  displayName: Delete or keep E2E Part 1 machine for debugging
-  type: string
-  default: delete
-  values:
-  - delete
-  - stop
-- name: E2EPart2AgentCleanup
-  displayName: Delete or keep E2E Part 2 machine for debugging
-  type: string
-  default: delete
-  values:
-  - delete
-  - stop
+  - name: DartLabEnvironment
+    displayName: DartLab Environment
+    type: string
+    default: Production
+    values:
+      - Production
+      - Staging
+  - name: E2EPart1AgentCleanup
+    displayName: Delete or keep E2E Part 1 machine for debugging
+    type: string
+    default: delete
+    values:
+      - delete
+      - stop
+  - name: E2EPart2AgentCleanup
+    displayName: Delete or keep E2E Part 2 machine for debugging
+    type: string
+    default: delete
+    values:
+      - delete
+      - stop
+  - name: ApexAgentCleanup
+    displayName: Delete or keep VS Apex test machine for debugging
+    type: string
+    default: stop
+    values:
+      - delete
+      - stop
 
 resources:
   pipelines:
-  - pipeline: DartLab
-    source: DartLab
-    branch: main
+    - pipeline: DartLab
+      source: DartLab
+      branch: main
```

Comment:
Why did you change the indenting on the entire file?

Reply:
I did a format document in VS Code by mistake which indented the entire file. I will revert that change in the next commit.